dags/nocp/README.md

```diff
@@ -1,14 +1,57 @@
-# Overview
+# Non-OCP workloads
 
-This directory contains all tasks, scripts, and default vars for openshift nightly performance builds
+Dags defined here doesn't need a OCP cluster and instead use an orchestration host to run the workloads.
+Dag definition file (airflow-kubernetes/dags/nocp/dag.py) defines these dags (i.e non OCP based applications).
+“OCM” is an example of such an application. Other non-OCP applications can utilize this DAG later.
 
 ## Structure
 
-* `dag.py`: The openshift-nightly DAG. If reading through the code, start here, as it outlines the precedence of the tasks in a DAG and wires everything else together.
+This directory contains all tasks, scripts, and default vars for openshift nightly performance builds
+
+* `dag.py`: The non-ocp DAG. If reading through the code, start here, as it outlines the precedence of the tasks in a DAG and wires everything else together.
 * `manifest.yaml`: The root-level configuration for all nightly DAGs. Children of 'platforms' are templated out to individual DAGs in airflow. More about [Manifests](./docs/manifest_and_releases.md).
 * `config`: Contains benchmark and installation configurations and variable defaults. [See docs/variables.doc for more](./docs/variables.md).
 * `docs`: Documentation for how the DAGs work and are wired together.
 * `models`: Contains two @dataclass classes used throughout. Honestly, the code is not strictly organized as there are plenty of classes defined in other directories.
 * `scripts`: Contains any executable run by an Airflow task
 * `tasks`: Contains python setup for Airflow Tasks
-* `util`: Contains utility functions for Airflow
\ No newline at end of file
+* `util`: Contains utility functions for Airflow
+
+# OCM DAG
+
+This DAG is used for OCM testing. Modules in this DAG -
+
+*1) config/benchmarks/ocm.json*
+* Secrets are fetched from vault.  All config variables defined in this file
+
+*2) manifest.yaml*
+* This manifest file defines ocm app and its auto schedule configuration (every Thursday 9am UTC). Autoschedule is enabled for only prod dag
+
+*3) tasks/benchmarks/nocp.py*
+* OCM tasks module. It defines “ocm-api-load” benchmark and “cleanup” tasks.
+* Ocm-api-load task executes run_ocm_benchmark.sh with “ocm-api-load” operation
+* cleanup task executes run_ocm_benchmark.sh with “cleanup” operation
```

Comment:
Defines the **ocm-api-load** benchmark and **cleanup** tasks.
* ocm-api-load task executes `run_ocm_benchmark.sh` with `ocm-api-load` operation
* cleanup task executes `run_ocm_benchmark.sh` with `cleanup` operation